airflow/configuration.py

```diff
@@ -228,6 +228,7 @@ def _validate_config_dependencies(self):
         is_executor_without_sqlite_support = self.get("core", "executor") not in (
             'DebugExecutor',
             'SequentialExecutor',
+            'MesosExecutor',
```

Comment:
This is asying MesosExecutor does support Sqlite. Given the workers are on remote nodes with no access to the .db file, and workers update state in the DB currently, I don't think this is true :)

Reply:
Yeah, thats not true. It does not.